### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by JoeDrubs** [aclap-dev/video-downloadhelper#659] [Help and Q&A]
### Your question

I don't know how to ticket so here

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.twitch.tv/videos/2148651103",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tls @ 000001f5c5f5ec40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c5f5ec40] Unknown error\r\n[tls @ 000001f5c6472d80] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472d80] Unknown error\r\n[tls @ 000001f5c634ac40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c634ac40] Unknown error\r\n[tls @ 000001f5c6472ac0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472ac0] Unknown error\r\n[tls @ 000001f5c64738c0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c64738c0] Unknown error\r\n[tls @ 000001f5c6472ac0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472ac0] Unknown error\r\n[tls @ 000001f5c64731c0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c64731c0] Unknown error\r\n[tls @ 000001f5c6473100] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6473100] Unknown error\r\n[tls @ 000001f5c6472f40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472f40] Unknown error\r\n[tls @ 000001f5c6473540] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6473540] Unknown error\r\n[tls @ 000001f5c6472ac0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472ac0] Unknown error\r\n[tls @ 000001f5c64732c0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c64732c0] Unknown error\r\n[tls @ 000001f5c6472f40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472f40] Unknown error\r\n[tls @ 000001f5c6472ac0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472ac0] Unknown error\r\n[tls @ 000001f5c6473380] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6473380] Unknown error\r\n[tls @ 000001f5c6472f40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472f40] Unknown error\r\n[tls @ 000001f5c6473100] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6473100] Unknown error\r\n[tls @ 000001f5c6472d80] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472d80] Unknown error\r\n[tls @ 000001f5c6472ac0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472ac0] Unknown error\r\n[tls @ 000001f5c6473640] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6473640] Unknown error\r\n[tls @ 000001f5c6472ac0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001f5c6472ac0] Unknown error\r\n[aac_adtstoasc @ 000001f5c6326240] Error parsing ADTS frame header!\r\n[mp4 @ 000001f5c6470780] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 000001f5c6399a00] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 000001f5c63689c0] Error muxing a packet\r\n",
    "id": "coapp_failure_8652ca49-314a-41f6-a841-d0f1b6950270",
    "downloadable_id": "downloadable_6724954459391613"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.  You didn't specify all of that information.  It is information that needs to be included in any problem report.

Your problem report gives a URL in the page_url tag.  I visited the page & got this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/efc3e3d3-c3f7-456f-b599-c05852fb0d94)

So I launched the download in VDH & everything seemed to be running fine:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/ae4e8e39-b278-4e54-9597-964c415b0c05)

You will not be surprised to learn that I had utterly no intention of downloading a video that is over 6 hours long, even though the download speed was quite good.  I stopped it shortly after I took that screenshot of the VDH progress.  The file I had was this:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8f1a4e7e-8099-43a5-b62c-c641697cd029)

And here it is playing in VLC:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/fbec5fa9-3f4e-4245-aa3d-d8642680b580)

You can see that the video was just fine.  You'll have to trust me that the audio was fine, too, although it seemed to have been recorded at a particularly low level.  I had to really crank up the volume to hear anything.  I didn't sit & watch this, just sampled it at intervals & it was fine.

You may not know "how to ticket," as you put it.  But if you're going to be posting here, you need to learn.  Click the Wiki link at the top of this page.  That will take you to what we currently have as a user manual.  Read it.  All of it.  There's a column of links to chapters down the right side of the page.  Click them all, including the ones at the bottom that are initially hidden.  You'll learn lots of useful things, including "how to ticket."

I don't know what to tell you about the error you encountered.  VDH had no problem downloading this stream for me.  It looks like you aren't running the same level of VDH as I am.  You need to.  You'll find the beta version of VDH as you read through the Wiki.
